### ISSUE TITLE
[AG-28] Mark as "internal" Node.js dependencies declared with GitHub urls from the organization

### ISSUE THREAD

**Issue by felipead** [toptal/license-cop#22]
When parsing Node.js dependencies inside `package.json`, we should mark as "internal" dependencies with git schemes pointing to the same organization we're generating the report for. In our specific case, this organization is Toptal, but we could generalize this detection to match the URL of the organization or repository passed as input to license-cop.

Examples:

```
"foo": "git+https://x-oauth-basic:12345678901234567890@github.com/acme/foo#v1.2.4"
"bar": "git+https://x-oauth-basic:12345678901234567890@github.com/acme/bar.git#v0.5.26"
"foo-bar": "git+https://x-oauth-basic:12345678901234567890@github.com/acme/foo-bar"
 "hello-world": "git+https://x-oauth-basic:12345678901234567890@github.com/acme/hello-world.git#df055e6",
```

Dependencies marked as "internal" should not be processed.
